src/databricks/labs/remorph/transpiler/transpile_engine.py

```diff
@@ -7,12 +9,39 @@
 
 class TranspileEngine(abc.ABC):
 
+    @classmethod
+    def load_engine(cls, transpiler: Path) -> TranspileEngine:
+        if str(transpiler) == "sqlglot":
```

Comment:
Add TODO here to modify this path once we have the SqlGlot plugin ready

Reply:
done